### PR TITLE
Feat/new participant view

### PR DESCRIPTION
**Pull request by MikeyManoguerra**
This sets up the front end to more easily build a VisitView, display a single participant's visits

### CODE REVIEW

**Comment by MikeyManoguerra** [comment]
#324

File: frontend/src/components/ParticipantInfo.js [hunk 1]
```diff
@@ -93,134 +85,18 @@ const ParticipantInfo = observer(() => {
...
-                  {insurers && insurers.length > 0 ? company.name : ""}
-                </MenuItem>
-              ))}
-            </Select>
-          </FormControl>
-        </Grid>
+        <ParticipantForm
+          insurers={insurers}
+          participantInfo={existingParticipant}
+          setRace={value => participantStore.setRace(value)}
```
Comment (ThrowsException) [inline frontend/src/components/ParticipantInfo.js:91]: Id recommend at this point just letting `ParticipantForm` have access to the `particpantStore` unless we expect these props to be reusuable with different store types.

File: frontend/src/components/ParticipantInfo.js [hunk 2]
```diff
@@ -287,11 +163,11 @@ const ParticipantInfo = observer(() => {
               onChange={e => participantStore.setVisitUrgency(e.target.value)}
               labelId="priority-level"
             >
-              <MenuItem value={"_1"}>1 (Lowest)</MenuItem>
-              <MenuItem value={"_2"}>2</MenuItem>
-              <MenuItem value={"_3"}>3</MenuItem>
-              <MenuItem value={"_4"}>4</MenuItem>
-              <MenuItem value={"_5"}>5 (Highest)</MenuItem>
+              {URGENCY_OPTIONS.map(urgency => (
```
Comment (ThrowsException) [inline frontend/src/components/ParticipantInfo.js:166]: 👍

File: frontend/src/components/QueueComponent/QueueTable/QueueTable.js [hunk 3]
```diff
@@ -13,10 +13,7 @@ import { QueueStoreContext } from "../../../stores/QueueStore"
 import { rootStoreContext } from "../../../stores/RootStore"
 import NotesDialog from "../../NotesDialog"
 import { useHistory } from "react-router-dom"
-import {
-  STATUS_OPTIONS,
-  URGENCY_OPTIONS,
-} from "../../../constants/GlobalConstants"
+import { STATUS_OPTIONS, URGENCY_OPTIONS } from "../../../constants"
```
Comment (ThrowsException) [inline frontend/src/components/QueueComponent/QueueTable/QueueTable.js:16]: 👍

File: frontend/src/components/WithSubmit.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            {submitText}
+          </PrevPointButton>
+        </Grid>
+      </Grid>
+    </Container>
+  )
+}
+
+WithSubmit.propTypes = {
+  component: PropTypes.func,
```
Comment (ThrowsException) [inline frontend/src/components/WithSubmit.js:38]: I think this would technically be a `PropTypes.element`  

```js
 // A React element.
  optionalElement: PropTypes.element,
```

**Comment by ThrowsException** [comment]
Overall +++1 on this PR @MikeyManoguerra just a couple small comments but id have no problem approving this as is if you want.

File: frontend/src/components/ParticipantInfo.js (hunk 1 above)
Comment (MikeyManoguerra) [reply frontend/src/components/ParticipantInfo.js:91]: ticket #321 describes a reusable handle change method, which would cut down on the number of function props. we wont be able to create methods for all the possible questions. Maybe that changes your opinion? this form doesn't need to be dynamically built but  most of the others will. 

TLDR this can stay stateful form but the dynamic ones probably shouldnt be

Comment (MikeyManoguerra) [reply frontend/src/components/ParticipantInfo.js:91]: I'm going to merge and revisit this later to keep the ball rolling
